region/lowernorfair/east.json

```diff
@@ -2655,63 +2716,663 @@
+                        {"canComeInCharged": {
+                          "framesRemaining": 180,
+                          "fromNode": 2,
+                          "shinesparkFrames": 0,
+                          "unusableTiles": 14
+                        }}
+                      ]}
+                    ]},
+                    {"heatFrames": 330},
+                    {"acidFrames": 210}
+                  ],
+                  "note": [
+                    "Jump directly from the top door's ramp, through the pirates, to the far bottom left of the acid filled room.",
+                    "Begin running from the top of the second slope and jump at the bottom of the third slope, bonking the large stalagtite in the ceiling.",
+                    "Kill the first pirate with Screw Attack or Charge+Plasma, and pass through the second pirate using a charged plasma shot.",
+                    "Alternatively, a very low speed shortcharge can run through both pirates.",
+                    "Enter the acid while aiming down to shrink Samus' hitbox.",
+                    "This strat works with all combinations of movement items and suits."
                   ]
                 }
               ]
             }
           ]
         },
         {
-          "from": 2,
+          "from": 3,
           "to": [
             {
               "id": 1,
               "strats": [
                 {
-                  "name": "Reverse Amphitheatre with Gravity",
+                  "name": "Amphitheatre Space Jump Acid Climb",
                   "notable": true,
                   "requires": [
                     "h_canNavigateHeatRooms",
+                    "canSuitlessLavaDive",
                     "Gravity",
-                    "canConsecutiveWalljump",
-                    {"heatFrames": 800},
-                    {"acidFrames": 650}
+                    "SpaceJump",
+                    {"or": [
+                      {"and": [
+                        {"heatFrames": 330},
+                        {"acidFrames": 330}
+                      ]},
+                      {"and": [
+                        "HiJump",
+                        {"heatFrames": 220},
+                        {"acidFrames": 220}
+                      ]}
+                    ]}
                   ],
                   "reusableRoomwideNotable": "Amphitheatre Reverse Acid Dive",
-                  "note": "Dive into the acid and get to the doorway high on the left wall."
+                  "note": "SpaceJump diagonally towards the door through the acid."
                 },
                 {
-                  "name": "Reverse Amphitheatre, Gravityless",
+                  "name": "Amphitheatre Acid Stutter Shinespark",
+                  "notable": true,
+                  "requires": [
+                    "h_heatProof",
+                    "Gravity",
+                    "canSuitlessLavaDive",
+                    "canStutterWaterShineCharge",
```

Comment:
I dont think this really fits. It's an incredibly short shinecharge, so I think itll be fine without it
```suggestion
```

Reply:
The shinecharge is easier as a stutterWaterShineCharge, but also if you can ShineCharge:13 then you don't need this.